packages/electron/README.md

```diff
@@ -0,0 +1,504 @@
+
+1. URL is not used - you can leave it as an empty string,
+1. You need to use `addBacktraceElectron` with the builder.
+
+For example, if you are initializing the SDK using the `initialize` function:
+
+```ts
+const client = BacktraceClient.initialize(
+    {
+        url: '',
+        name: 'My Renderer App',
+        version: '0.0.1',
+    },
+    (builder) => addBacktraceElectron(builder),
+);
+```
+
+Or, if you are using the `build` function:
+
+```ts
+const builder = BacktraceClient.builder({
+    url: '',
+    name: 'My Renderer App',
+    version: '0.0.1',
+});
+
+addBacktraceElectron(builder);
+
+const client = builder.build();
+```
+
+#### Synchronized initialization
+
+When adding Electron to the renderer SDK, data is exchanged with the main process to ensure same session IDs are used.
+By default, this is done asynchronously as soon as possible. This may however be not enough in case of startup crashes
+just after the SDK initialization, because the data may not be exchanged yet.
+
+In those cases, you can use `synchronize` option in `addBacktraceBuilder`. Note, that if you do not ensure that the SDK
+in the main process is initialized by the time you call `addBacktraceBuilder`, **your application may hang with a white
+screen**. This is due to the synchronous nature of the IPC, that is blocking the renderer process until it receives a
+response.
+
+To enable synchronous initialization:
+
+```ts
+const client = BacktraceClient.initialize(
+    {
+        url: '',
+        name: 'My Renderer App',
+        version: '0.0.1',
+    },
+    (builder) => addBacktraceElectron(builder, { synchronous: true }),
+);
+```
+
+### Upload source maps
+
+Client-side error reports are based on minified code. Upload source maps and source code to resolve minified code to
+your original source identifiers.
+
+[(Source Map feature documentation)](https://docs.saucelabs.com/error-reporting/platform-integrations/source-map/)
+
+<? TBD: Link to source upload doc ?>
+
+## Error Reporting Features
+
+### Attributes
+
+Custom attributes are key-value pairs that can be added to your error reports. They are used in report aggregation,
+sorting and filtering, can provide better contextual data for an error, and much more. They are foundational to many of
+the advanced Backtrace features detailed in
+[Error Reporting documentation](https://docs.saucelabs.com/error-reporting/getting-started/). By default attributes such
+as application name and version are populated automatically based on your package.json information. If Backtrace cannot
+find them, you need to provide them manually via userAttributes attributes.
+
+There are several places where attributes can be added, modified or deleted.
+
+#### Attach attributes object to BacktraceClient
+
+It is possible to include an attributes object during [BacktraceClient](#backtraceclient) initialization. This list of
+attributes will be included with every error report, referred to as global attributes.
+
+```ts
```

Comment:
should we point this is the main process?

Reply:
Done.